sdk/storage/azure_storage_blob/Cargo.toml

```diff
@@ -1,11 +1,16 @@
 [package]
 name = "azure_storage_blob"
 version = "0.1.0"
+description = "Microsoft Azure Blob Storage client library for Rust"
+readme = "README.md"
 authors.workspace = true
 edition.workspace = true
 license.workspace = true
 repository.workspace = true
 rust-version.workspace = true
+homepage = "https://github.com/azure/azure-sdk-for-rust"
+documentation = "https://docs.rs/azure_storage_blob"
+keywords = ["sdk", "azure", "storage", "blob", "blobs"]
```

Comment:
You should add `categories` as well. See the other crates' `Cargo.toml` files as well.